sky/services/dynamic/dynamic_service_dylib.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/services/dynamic/dynamic_service.h"
+
+#include <cassert>
```

Comment:
you should probably use <assert.h>. the assert header defines a macro so the namespacing functionality that most <cfoo> headers provide is extra useless here.